app/controllers/jobs_controller.rb

```diff
@@ -11,8 +11,20 @@ def show
   def create
     @job = JobPosting.new(job_params)
 
+    # Collect and cache information required for future moderation actions
+    @job.attributes = {
+      user_ip: request.env['REMOTE_ADDR'],
+      user_agent: request.env['HTTP_USER_AGENT'],
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.